src/Simplex/Chat/Protocol.hs

```diff
@@ -243,13 +243,22 @@ instance ToJSON SharedMsgId where
   toJSON = strToJSON
   toEncoding = strToJEncoding
 
+data MsgScope
+  = MSGroup
+  | MSMember {memberId :: MemberId} -- Admins can use any member id; members can use only their own id
```

Comment:
how is it possible to understand from this name that scope is conversation with admins